paasta_tools/utils.py

```diff
@@ -1657,3 +1661,27 @@ def mean(iterable):
     Returns the average value of an iterable
     """
     return sum(iterable) / len(iterable)
+
+
+def prompt_pick_one(sequence, choosing):
+    if not sys.stdin.isatty():
+        sys.stderr.write('No {choosing} specified and no TTY present to ask.'
+                         ' Please specify a {choosing} using the cli.\n'.format(choosing=choosing))
+        sys.exit(1)
+
+    QUIT_ACTION = 'Quit'
+    global_actions = [QUIT_ACTION]
+    choices = [(item, item) for item in sequence]
+
+    chooser = choice.Menu(choices, global_actions=global_actions)
+    chooser.title = 'Please pick a {choosing} from the choices below:'.format(choosing=choosing)
+    try:
+        result = chooser.ask()
+    except KeyboardInterrupt:
+        sys.stdout.write('\n')
+        sys.exit(1)
+
+    if isinstance(result, tuple) and result[1] == QUIT_ACTION:
```

Comment:
Are there situations where the results is a tuple but not Quit action?

I don't want to give callers tuples if they are expecting a singleton